README.md

```diff
@@ -6,7 +6,8 @@ Install:
 
 - Python 3, at least Python 3.6.
```

Comment:
With the `:=` syntax used in catalog.py, you'll need to update this to be at least Python 3.8.

Reply:
darn good catch! we want to keep it to python 3.6 so it works in scripts, so let me downgrade.